ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -241,7 +241,7 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
-    private(set) var style: Style = defaultStyle
+    @objc dynamic private(set) var style: Style = defaultStyle
```

Comment:
is there a reason why this needs to be dynamic?

Reply:
added comment, but it's so we can do KVO on it